<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserMainWindow</class>
 <widget class="QMainWindow" name="UserMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>登录窗口</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="planeLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="figures.qrc">:/img/source/figures/plane.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>5</y>
      <width>270</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>航班管理系统（客户端）</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="backBtn">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退出登录</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>12</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>UserName</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>修改密码</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>退出登录</string>
     </property>
    </item>
   </widget>
   <widget class="Line" name="horizontalline_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>800</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="flightQueryBtn">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>70</y>
      <width>130</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>航班查询</string>
    </property>
    <property name="icon">
     <iconset resource="figures.qrc">
      <normaloff>:/img/source/figures/flightQuery.png</normaloff>:/img/source/figures/flightQuery.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="myOrdersBtn">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>150</y>
      <width>130</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>我的订单</string>
    </property>
    <property name="icon">
     <iconset resource="figures.qrc">
      <normaloff>:/img/source/figures/myOrders.png</normaloff>:/img/source/figures/myOrders.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="myFavoritesBtn">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>230</y>
      <width>131</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>我的收藏</string>
    </property>
    <property name="icon">
     <iconset resource="figures.qrc">
      <normaloff>:/img/source/figures/myFavorites.png</normaloff>:/img/source/figures/myFavorites.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="personalCenterBtn">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>310</y>
      <width>130</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>个人中心</string>
    </property>
    <property name="icon">
     <iconset resource="figures.qrc">
      <normaloff>:/img/source/figures/personalCenter.png</normaloff>:/img/source/figures/personalCenter.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="Line" name="Verticalline">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>20</width>
      <height>550</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>650</width>
      <height>550</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QLabel" name="label_1">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>100</y>
        <width>80</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Page 1</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>100</y>
        <width>80</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Page 2</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>100</y>
        <width>80</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Page 3</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="avatarLabel">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>头像</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="accountLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>80</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>账号：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="accountEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>140</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="genderLabel">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>130</y>
        <width>80</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <property name="text">
       <string>性别：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="phoneLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>200</y>
        <width>80</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>手机号：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="phoneEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>210</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="emailLabel">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>200</y>
        <width>80</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>邮箱：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="emailEdit">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>210</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="balanceLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>270</y>
        <width>90</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>账户余额：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="balanceEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>280</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="Line" name="horizontalline_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>350</y>
        <width>660</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="editInfoBtn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>430</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #156080;
    color: white;
    border: 1px solid #ababab;
    border-radius: 12px;
}</string>
      </property>
      <property name="text">
       <string>编辑个人信息</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rechargeBtn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>430</y>
        <width>160</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #156080;
    color: white;
    border: 1px solid #ababab;
    border-radius: 12px;
}</string>
      </property>
      <property name="text">
       <string>充值</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cancelAccountBtn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>430</y>
        <width>160</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: red;
    color: white;
    border: 1px solid #ababab;
    border-radius: 12px;
}</string>
      </property>
      <property name="text">
       <string>注销账号</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="figures.qrc"/>
 </resources>
 <connections/>
</ui>
